src/data/equiv/option.lean

```diff
@@ -0,0 +1,72 @@
+/-
+Copyright (c) 2021 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author: Eric Wieser
+-/
+import data.equiv.basic
+import control.equiv_functor
+
+/-!
+# Equivalences for `option α`
```

Comment:
Note that naively I would expect to find in this file a definition that turns `equiv X Y` into `equiv (option X) (option Y)`.
Surely this somewhere else in mathlib. Could you please add a pointer to it in this module docstring?

Reply:
Added